Makefile

```diff
@@ -3,47 +3,54 @@
 PREFIX := /usr
 DESTDIR :=
 
-ifndef VERSION
-MAJORVERSION := 8
-MINORVERSION ?= $(shell git rev-list --count master)
-endif
-VERSION := ${MAJORVERSION}.${MINORVERSION}
+
+MAJORVERSION$(VERSION) := 8
+MINORVERSION$(VERSION) != git rev-list --count master
+VERSION ?= ${MAJORVERSION}.${MINORVERSION}
 
 LDFLAGS := -ldflags '-s -w -X main.version=${VERSION}'
 ARCH := $(shell uname -m)
 PKGNAME := yay
 BINNAME := yay
 PACKAGE := ${PKGNAME}_${VERSION}_${ARCH}
 
-export GOPATH=$(shell pwd)/.go
+GOPATH=$(shell pwd)/.go
```

Comment:
`export` isn't valid outside of GNU make, but it's unnecessary in either. This should be propagated into any subshells, and I think this behavior is the same for GNU make and BSD make.

Reply:
Interesting, let me double check and I'll add it back in if there's not a standard way to do this. Thanks!

Reply:
Could be a darwin thing (or super outdated bash thing). I think it's safe to remove it

Reply:
Odd, I thought I tested this but apparently I'm completely wrong about the behavior. export does work in both, it's the `$(shell pwd)` that's not working in bsd make for some reason. I've updated it and tested with both `gmake` and `bmake`: the absolute path is correct in both now.